doozer/doozerlib/runtime.py

```diff
@@ -507,7 +507,10 @@ def filter_disabled(n, d):
             elif self.group_config.arches_override:  # Allow arches_override in group.yaml to temporarily override GA architectures
                 self.arches = self.group_config.arches_override
             else:
-                self.arches = self.group_config.get('arches', ['x86_64'])
+                if self.build_system == 'brew':
+                    self.arches = self.group_config.get('arches', ['x86_64'])
+                else:
+                    self.arches = self.group_config.get('konflux', {}).get('arches', ['x86_64'])
```

Comment:
thinking if this is going to be a recurring pattern we can do something like this in runtime

```python
if self.build_system == "brew":
   group_config_without_konflux = {k:v for k, v in self.group_config if k != 'konflux' }
   # override konflux specific settings
   self.group_config = group_config_without_konflux.update(self.group_config.get('konflux', {}))
```

Reply:
Done [here](https://github.com/openshift-eng/art-tools/pull/1502/files#diff-1406051f1a04a744fb77f4c5faaac7f4b33f803c82506229b6e02bc7499e744fR254-R255)